app/services/notifications/discord_notifier.rb

```diff
@@ -0,0 +1,82 @@
+module Notifications
+  class DiscordNotifier
```

Comment:
please move this to `app/integrations/discord`. No discord specific code should live in `app/services/`

Reply:
Also sorry, I didn't get what do you have in mind with this class. What exactly is a discord notification?